### PR TITLE
Js-oop frogger game

### PR DESCRIPTION
**Pull request by HelenGreent**
### Object Oriented JS (frogger)

[Demo](https://helengreent.github.io/kottans-frogger-game) |
[Code base](https://github.com/HelenGreent/kottans-frogger-game)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by HelenGreent** [comment]
re-open

**Comment by OleksiyRudenko** [comment]
We still have irrelevant commits that add code from a different task here
image

Check the bot's comments in these regards and fix. Do not ignore bot's suggestion to seek help and ask questions in STudents' chat.

**Comment by OleksiyRudenko** [comment]
This will delete your earlier submission.
On commecrial projects this is something we want to avoid.
image

File: submissions/HelenGreent/js-oop/app.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  this.y = y;
+  this.sprite = "images/char-boy.png";
+};
+
+Player.prototype.update = function () {
+  this.checkCollision();
+};
+
+Player.prototype.checkCollision = function () {
+  allEnemies.forEach(function (enemy) {
```
Comment (OleksiyRudenko) [inline submissions/HelenGreent/js-oop/app.js:74]: >     * [ ] classes do not refer to any global variables, like global variable `player`, which is an instance of `Player` class
>      (referring to global constants and globals provided by the gaming platform like `Resources` is OK);
>      Hint: pass `Player` instance as an argument to every enemy

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@HelenGreent a pretty decent job done.
Although one improtant issue yet to fix.
And we do not want to remove any code outside the scope of current PR (see the comment above)

**Comment by HelenGreent** [comment]
Understand it, but I didn’t find a better way to fix my issue than deleting tiny-js-world from an oop-exercise feature in the current PR. I was thinking about changing history but it didn’t change the mistake

**Comment by OleksiyRudenko** [comment]
Open a new PR for this, following the contribution instructions.
Learn to fix this type of git-wrokflow mistakes. You will need this on a commercial project. Constribution FAQ section offers a relevant learning material.

**Comment by HelenGreent** [comment]
> Open a new PR for this, following the contribution instructions. Learn to fix this type of git-wrokflow mistakes. You will need this on a commercial project. Constribution FAQ section offers a relevant learning material.

Should I use "git revert" in this situation and push a new commit which reverts the pushed commits?

**Comment by HelenGreent** [comment]
> Open a new PR for this, following the contribution instructions. Learn to fix this type of git-wrokflow mistakes. You will need this on a commercial project. Constribution FAQ section offers a relevant learning material.

link to new PR:
https://github.com/kottans/frontend-2022-homeworks/pull/748

**Pull request closed without merging**